pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -344,6 +353,30 @@ func coreDNSLogLevel(dns *operatorv1.DNS) string {
 	return "class error"
 }
 
+// coreDNSCache reads the TTL values set in spec.cache and returns integer representations of those values in seconds.
+// This returns uint32 values because CoreDNS won't allow setting the maxTTLs to 0 and there's no use case for setting a
+// negative maxTTL value.
+func coreDNSCache(dns *operatorv1.DNS) (positiveTTL, negativeTTL uint32) {
+	var p, n uint32
```

Comment:
As you already have named return parameters `positiveTTL` and `negativeTTL`, you can just use them instead of creating new variables.

Reply:
Fixed!